_includes/master/felix-typha-tls-howto.md

```diff
@@ -0,0 +1,58 @@
+
+Here is an example of how you can secure the Felix-Typha communications in your
+cluster:
+
+1.  Choose a Certificate Authority, or set up your own.
+
+1.  Obtain or generate the following leaf certificates, signed by that
+    authority, and corresponding keys:
+
+	-  A certificate for each Felix with Common Name 'typha-client' and
```

Comment:
These bullets might be indented too far - if so, they'll render as a pre-formatted block instead of bullets.  In any case, they're inconsistent with the bullets below so probably worth syncing up.